cmd/skaffold/app/cmd/run.go

```diff
@@ -49,6 +49,9 @@ func doRun(ctx context.Context, out io.Writer) error {
 		err = r.DeployAndLog(ctx, out, bRes)
 		if err == nil {
 			tips.PrintForRun(out, opts)
+			if opts.Cleanup {
```

Comment:
shd we cleanup only if `--tail` is true?

Reply:
I feel like it's fine how it is. Having flags depend on other flags is a bit clunky imo, and would also make the docs for this flag more cumbersome/confusing

Reply:
Um except Cleanup does a delete of the resources, so without `--tail` you'll deploy and then immediately delete newly-deployed resources.  Although that may be literally what's specified, it doesn't seem like a great UX :-)